.github/workflows/mirror-functions.yaml

```diff
@@ -0,0 +1,45 @@
+name: mirror-functions
+
+on:
+  push:
+    branches:
+      - "*"
+  schedule:
+    - cron: "0 18 * * *"
+
+jobs:
+  mirror:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up gcloud
+        uses: GoogleCloudPlatform/github-actions/setup-gcloud@master
+        with:
+          version: "275.0.0"
+          service_account_email: ${{ secrets.GCP_SA_EMAIL }}
+          service_account_key: ${{ secrets.GCP_SA_KEY }}
+      - run: gcloud auth configure-docker
+      - name: Mirror kpt-functions images
+        env:
+          PROJECT_ID: kpt-functions
+        run: |
+          declare -A kptFunctionsImages=(["read-yaml"]="read-yaml" ["write-yaml"]="write-yaml" ["gatekeeper-validate"]="policy-controller-validate")
```

Comment:
create a bash script in top-level scripts dir

Reply:
Done